internal/util/util/translate.go

```diff
@@ -0,0 +1,44 @@
+package util
```

Comment:
`util` isn't a great package name, since it is only used in `server` I'd move this file there and rename the function to `renameErrors`

Reply:
done